Minio/DataModel/Replication/ReplicationConfiguration.cs

```diff
@@ -54,7 +53,7 @@ public ReplicationConfiguration(string role, Collection<ReplicationRule> rules)
 
     [XmlElement("Role")] public string Role { get; set; }
 
-    [XmlElement("Rule")] public Collection<ReplicationRule> Rules { get; set; }
+    [XmlElement("Rule")] public ICollection<ReplicationRule> Rules { get; set; }
```

Comment:
@ebozduman XML needs a concrete type. Setting this to an interface might break things?